jest.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = {
     "transform": {
         "^.+\\.[t|j]sx?$": "babel-jest"
     },
-    "testEnvironment": "jsdom"
+    "testEnvironment": "jsdom",
+    "transformIgnorePatterns": [],
```

Comment:
What failed for you to need this?

Reply:
By default, Jest ignores `node_modules`.
Apparently `jsdom` doesn't properly support es6's import/export properly, so this allows Jest to use that.
More importantly, in "production" we don't transpile; We use the code as is (Chrome supports import/export). This makes the tests more loyal to the current production environment of this.